packages/office-ui-fabric-react/src/utilities/positioning.ts

```diff
@@ -20,6 +21,23 @@ export enum Position {
   end = 3
 }
 
+export enum CalloutLinkType {
```

Comment:
I initially did define it in Callout's props file, but then it is also being in positioning.ts.

Is there a way to export an imported module? That way I could export it in Callout.Props.ts while using it and it could make it to documentation.